### PR TITLE
Purges zombie code

### PR DESCRIPTION
**Pull request by DominikPanic**
It had to be done

### CODE REVIEW

**Comment by Ghommie** [comment]
Good riddance. I gave this a look a month or two ago ~~after breaking up some things~~, It's badly done both by design and code, so just better to port some at a later date.

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -17,8 +17,7 @@
 	life_tick++
 
 	voice = GetVoice()
-	if(stat == DEAD && iszombie(src) && regenZ)
-		handle_organs()
+	if(stat == DEAD)
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/human/life.dm]: This is conflict with Spooky's life PR. Keep an eye on it.

Comment (DominikPanic) [reply code/modules/mob/living/carbon/human/life.dm]: Also it will definitely conflict with all of my attackby PRs

Comment (DominikPanic) [reply code/modules/mob/living/carbon/human/life.dm]: And done.

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -18,11 +18,9 @@
 
 	voice = GetVoice()
 	if(stat == DEAD)
-		if(iszombie(src) && regenZ)
-			handle_organs()
-			return
 		if(!check_tod())
 			SSmobs.stop_processing(src)
+		return
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/human/life.dm]:
```suggestion

```

**Pull request merged**